packages/core/src/components/navbar/navbar.md

```diff
@@ -10,7 +10,11 @@ Enable the `fixedToTop` prop to attach a navbar to the top of the viewport using
 `position: fixed; top: 0;`. This is so-called "sticky" behavior: the navbar
 stays at the top of the screen as the user scrolls through the document.
```

Comment:
this sentence is no longer correct if the component offers actual `sticky` support.